LondonBritishLibrary/orient/BLorient764.xml

```diff
@@ -25,26 +25,452 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.19">
+<locus from="167r"/>
+<title type="complete" xml:lang="en" ref="LIT4838CollectioM#Chapter19">Asceticon of Pachomius</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.20">
+<locus from="168r"/>
+<title type="complete" xml:lang="gez" ref="LIT4838CollectioM#Chapter20">ግብረ፡ አበው፡ ዘመነኮሳት፡</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.21">
+<locus from="168v"/>
+<title type="complete" xml:lang="gez" ref="LIT4838CollectioM#Chapter21">ለመነኮሳት፡ ምግባራት፡ ለእለ፡ ይነብሩ፡ ውስተ፡ ማኅበር፡ መነኮሳት፡ ወራስያነ፡ እግዚአብሔር፡</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.22">
+<locus from="172r"/>
+<title type="complete"  xml:lang="en" ref="LIT4838CollectioM#Chapter22">Ephraim Syrus</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.23">
+<locus from="173r"/>
+<title type="complete" xml:lang="en" ref="LIT4838CollectioM#Chapter23">The same, <foreign xml:lang="gez">በእንተ፡ ሕፃናት፡</foreign></title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.24">
+<locus from="175v"/>
+<title type="complete" xml:lang="gez" ref="LIT4838CollectioM#Chapter24">አበይኖ፡ ለእለ፡ ይነብሩ፡ ውስተ፡ ማኅደር፡</title>
+<note>Incomplete in the middle.</note>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.25">
+<locus from="178v"/>
+<title type="complete" xml:lang="en" ref="LIT4838CollectioM#Chapter25">Ammon</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.26">
+<locus from="179v"/>
+<title type="complete" xml:lang="gez" ref="LIT4838CollectioM#Chapter26">በእንተ፡ ፍቅረ፡ ወርቅ፡ ወብሩር፡</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.27">
+<locus from="180r"/>
+<title type="complete" xml:lang="gez" ref="LIT4838CollectioM#Chapter27">በእንተ፡ ትውጽሕት፡</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.28">
+<locus from="181r"/>
+<title type="complete" xml:lang="gez" ref="LIT4838CollectioM#Chapter28">ነገራተ፡ አእሩግ፡ ዘተናገሩ፡</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.29">
+<locus from="182r"/>
+<title type="complete" ref="LIT4838CollectioM#Chapter29">John Chrysostom, <foreign xml:lang="gez">ዘዮሐንስ፡ ጳጳስ፡ ዘቈስጠንጢኖጶሊስ፡ በእንተ፡ አርምሞ፡ ዘተናገረ፡</foreign></title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.30">
+<locus from="182v"/>
+<title type="complete" xml:lang="en" ref="LIT4838CollectioM#Chapter30">Another extract from Chrysostom</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.31">
+<locus from="183v"/>
+<title type="complete" xml:lang="en" ref="LIT4838CollectioM#Chapter31">Peter, the father of Ephraim</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.32">
+<locus from="184v"/>
+<title type="complete" xml:lang="en" ref="LIT4838CollectioM#Chapter32">Anonymous</title>
```

Comment:
Anonymous discourse? and same below

Reply:
I should probably be less literal when transcribing Wright